Source/Scene/Expression.js

```diff
@@ -43,6 +43,19 @@ define([
         }
     };
 
+    var unaryFunctions = {
+        abs : Math.abs,
+        sqrt : Math.sqrt,
+        cos : Math.cos,
+        sin : Math.sin,
+        tan : Math.tan,
+        acos : Math.acos,
+        asin : Math.asin,
+        atan : Math.atan,
+        radians : CesiumMath.toRadians(),
+        degrees : CesiumMath.toDegrees()
```

Comment:
Could  it be something wrong with  the way I'm trying to  include CesiumMath? I included the core math package in the first define, then CesiumMath in the second, though that could be wrong

![could](https://cloud.githubusercontent.com/assets/10562115/20444954/29759684-ada1-11e6-9faa-40603623e65a.png)